### ISSUE TITLE
📝 Correction | Mailbox.org should be labeled as green for accepting mail-in cash

### ISSUE THREAD

**Issue by techlore-official** [privacytools/privacytools.io#2008]
## Description

Here is the best case email scenario per the privacy criteria listed on PTIO:
*Accepts Bitcoin, cash, and other forms of cryptocurrency and/or anonymous payment options (gift cards, etc.)*

Mailbox.org offers mail-in cash, which in many ways is a more private form of payment than Bitcoin (Which services like Protonmail received a "green" status for). I'm not sure I agree with the implication that Bitcoin is more private than cash based on the coloring scheme, and believe Mailbox.org should be given a matching color.

## Why I am making the suggestion

I was navigating email providers and noticed mailbox.org received a yellow payment score, and didn't feel it was fair considering they offer arguably one of the best forms of payment compared to other services. Pretty simple 🤷‍♂️

## My connection with the software

No connection

**Comment by dngray** [comment]
This is a good point. I think in the past we were giving green to payment systems that supported cryptocurrency.

Including cash is good, the only counter view I can see to that is they only do so for German users iirc and not internationally.

**Comment by thansk** [comment]
This is true for Posteo too. They too accept cash but not cryptocurrencies. I don't know about Mailbox, but Posteo has no requirements on where the cash comes from.

**Comment by dngray** [comment]
I'm not so sure that cash has the same anonymity that cryptocurrencies can.

If you're mailing it, there's still going to be postmark stamps? Do they accept cash from international customers?

Of course things like Bitcoin are not anonymous, if you just buy them and spend them, however if you filter them through another currency like Monero, then it is.
